apply dj-rest-auth

Ridealist opened the pull request:
<!-- Hello! Thanks for submitting a PR. To help make things go a bit more
     smoothly we would appreciate that you go through this template. -->

## Features
* apply dj-rest-auth package


### Description
* requirements에 dj-rest-auth == 2.2.2 추가
* base.py/local apps에 반영
* CHANGELOG에 내용 추가

<!-- Good things to put here include: reasoning for the change (please link
     any relevant issues!), any noteworthy (or hacky) choices to be aware of,
     or what the problem resolved here looked like. You can write in Korean or English as you want  -->


### Checklist - did you ...

<!-- If any of the following items aren't relevant for your contribution
     please still tick them so we know you've gone through the checklist.

    All user-facing changes should get an entry. Otherwise, signal to us
    this should get the magical label to silence the CHANGELOG entry check.
    Tests are required for bugfixes and new features. Documentation changes
    are necessary for formatting and most enhancement changes. -->

- [x] Add a CHANGELOG entry if necessary?
- [x] Add / update tests if necessary?
- [x] Add new / update outdated documentation?

Ridealist committed:
apply dj-rest-auth

hoetaek commented on config/settings/base.py:
```diff
@@ -79,6 +79,8 @@
 
 LOCAL_APPS = [
     "kkanbu.users",
+    "dj_rest_auth",
```
LOCAL_APPS가 아니라 THIRD_PARTY_APPS에 넣어두어야 할 것 같습니다

Ridealist replied:
base.py LOCAL_APPS 부분에 # Your stuff: custom apps go here 이렇게 나와있어서 여기에 세팅했는데,
이걸 내가 커스텀한 앱이라고 해석하면 THIRD_PARTY_APPS에 넣는 방법도 일리가 있네요.

hoetaek replied:
코드 작동하는 데에는 문제가 없지만 pip install 해서 설치한 것이라서 third party app에 해당됩니다~ 제가 직접 startapp 한 것은 local app이고요

Ridealist replied:
아 그런 의미였군요. 내 의견 감사해요. 수정해 놓을게요:)

hoetaek replied:
넵!

Ridealist committed:
Update README.rst

Docker 실행 방법 상세 설명 추가

Ridealist committed:
add docker command

Ridealist committed:
readme edit

Ridealist committed:
Update README.rst

docker-compose -f local.yml up 이 commad를 통해 장고 웹사이트는 이미 실행이 되네요.^^;;
docker-compose -f local.yml run --rm python manage.py runserver 명령은 취소합니다!

Ridealist committed:
Update CHANGELOG.md

THIRD_PARTY_APPS 변경

Ridealist committed:
Update base.py

THIRD_PARTY_APPS 변경

hoetaek reviewed (approved):
감사합니다!

teacherSsamko reviewed (approved):
Good Start 👍😎
